### ISSUE TITLE
"Give a Spare Part" deathrattles don't always work when copied by Unearthed Raptor

### ISSUE THREAD

**Issue by jleclanche** [HearthSim/hs-bugs#268]
Deathrattles that give a spare part cannot be copied by Unearthed Raptor because of the random entourage mechanic.

https://www.reddit.com/r/hearthstone/comments/3thdcg/unearthed_raptor_clockwork_gnome_bug/

However, if the Clockwork Gnome dies first, then the Unearthed Raptor can give you a spare part as normal.

* https://www.youtube.com/watch?v=NYMw8vuDeU0

**Comment by PattuX** [comment]
"I tested this a few times vs the innkeeper and it's very easily repeatable. The raptor's death rattle wont go off as long as the clockwork is alive. If you kill off the clockwork the raptor's deathrattle functions correctly. Toshley works the same as the clockwork gnome. Mechanical Yeti seems to work fine either dead or alive."
From the reddit threat, so I guess there has to be some other reasoning behind this.

**Comment by Patashu** [comment]
Should this be a new issue or part of its own issue? Player B plays Unearthed Raptor on Mechanical Yeti, Player A gets a copy by Mirror Entity, Player A's copy does not give spare parts, Player B's does:

http://us.battle.net/hearthstone/en/forum/topic/19974747281

**Comment by jleclanche** [comment]
Can you reproduce this with a different target? If the bug is "Copied Unearthed Raptor does not carry copied deathrattles", that's a separate bug.

**Comment by Xinhuan** [comment]
http://us.battle.net/hearthstone/en/forum/topic/19973697317

Mechanical Yeti appears to be bugged in the same manner, this report says that Yeti was alive, and the Raptor (with 2 copies of the Deathrattle due to Brann) didn't trigger.

**Comment by PattuX** [comment]
Just played a game, where I decided to misplay for the sake of science.
I played Raptor on my Nerubian Egg, which got copied by Mirror Entity. The result was this: http://imgur.com/nqWJKx0 
However, his Raptor summoned the Nerubian correctly. (Unfortunatelly no screenshot :/).
But somehow, after I copied my Egg once again, his game was propably frozen, as he didn't play any cards, roped every turn but still emoted.
